sql/driver/driver_test.go

```diff
@@ -279,7 +294,10 @@ func TestInsecure(t *testing.T) {
 // before committing the transaction, start another transaction in
 // another thread to update the same row. The other transaction succeeds,
 // resulting in the first transaction failing.
-func TestOverlappingTransactions(t *testing.T) {
+// ***** This test has been disabled ********
+// TODO(vivek): Fix/Rewrite this flaky test.
+// Fix issue https://github.com/cockroachdb/cockroach/issues/2366
+func DisabledTestOverlappingTransactions(t *testing.T) {
```

Comment:
I wont merge this in before this test is fixed. I just want this PR to be ready so when the test it fixed I can merge it.